client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -86,6 +94,16 @@ const ConnectedSingleSiteJetpack = connectOptions(
 			<div>
 				<SidebarNavigation />
 				<CurrentTheme siteId={ siteId } />
+				{
+					( config.isEnabled( 'jetpack/pijp' ) && ! requestingSitePlans && ! hasUnlimitedPremiumThemes ) && <Banner
+						plan={ PLAN_JETPACK_PREMIUM }
+						title={ translate( 'Access all our premium themes with our Business plan!' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Access all our premium themes with our Premium and Business plans!' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).